### DISCUSSION TITLE
https://ckeditor.com/ckeditor-5/builder/

### DISCUSSION THREAD

**Discussion by Lancilllotto** [community/community#132445] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

regarding the codes you exposed to create an html editor for my website. The procedures are not clear. I saw the codes but what should I do? should I take photos with my cell phone, print them and send them to friends by post as postcards with a photo of the parking lot at my house? any procedural logic is missing.

https://ckeditor.com/ckeditor-5/builder/

**Comment by LiteBrite82** [comment]
Hi @Lancilllotto ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/ckeditor/ckeditor5). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Lancilllotto** [comment]
ok many thanks... i  see   the link

Il giorno mer 17 lug 2024 alle ore 00:28 LiteBrite82 <
***@***.***> ha scritto:
